celery/worker/request.py

```diff
@@ -502,6 +506,19 @@ def group(self):
         # by parent process
         return self.request_dict.get('group')
 
+    @cached_property
+    def request(self):
```

Comment:
The property name is not very meaningful.
`some_request.request` does not explain what the property returns.
We'll need to rename it.
Furthermore, should this property be public? Is there a reason to expose this to Celery's users?
And one last thing, we need a docstring for this property.